src/components/bs5/searchInput/searchInput.scss

```diff
@@ -98,8 +98,10 @@
 
                     &:hover {
                         background-color: var(--#{$prefix}site-search-suggestions-hover);
-                        margin: 0 -1rem;
-                        padding: 0 1rem;
+                        margin-left: -1rem;
+                        margin-right: -1rem;
+                        padding-right: 1rem;
+                        padding-left: 1rem;
```

Comment:
Hey @ghazal-salehi, is it possible to move these margins away from the hover state?  

Thinking is margins and padding should be consistent across all states, so if we set them on LI's default state and just let it inherit to all others?

Reply:
Happy for you guys to take over the styling of this feature

Reply:
Thanks Mitch. Please do.

Reply:
Hi @stvp-qld. I removed the padding and margin rules for hover state. Now the search input dropdown style matches the site of the global search in the header.

Reply:
Intended:
<img width="428" alt="Screenshot 2024-06-24 at 11 37 28 am" src="https://github.com/qld-gov-au/qgds-bootstrap5/assets/122584447/2bd22756-57da-47b0-96bd-3232fedbbc05">

without margin/padding on hover element:
<img width="413" alt="Screenshot 2024-06-24 at 11 36 33 am" src="https://github.com/qld-gov-au/qgds-bootstrap5/assets/122584447/ac8ee35b-083e-4029-8d26-091abc6f019c">

we just have to make sure is reflected elsewhere and can inherit